src/components/media_manager/CMakeLists.txt

```diff
@@ -67,6 +67,10 @@ set(LIBRARIES
   Utils
 )
 
+if(${OS} STREQUAL "qnx")
+  list(APPEND LIBRARIES socket)
```

Comment:
@BSolonenko can we try to use cmake generators? Please check if it compatible with current cmake version

Reply:
@AKalinich-Luxoft  What do you mean?